go.sum

```diff
@@ -287,6 +283,8 @@ github.com/valyala/bytebufferpool v1.0.0/go.mod h1:6bBcMArwyJ5K/AmCkWv1jt77kVWyC
 github.com/valyala/fasttemplate v1.2.1/go.mod h1:KHLXt3tVN2HBp8eijSv/kGJopbvo7S+qRAEEKiv+SiQ=
 github.com/valyala/fasttemplate v1.2.2 h1:lxLXG0uE3Qnshl9QyaK6XJxMXlQZELvChBOCmQD0Loo=
 github.com/valyala/fasttemplate v1.2.2/go.mod h1:KHLXt3tVN2HBp8eijSv/kGJopbvo7S+qRAEEKiv+SiQ=
+github.com/xlab/tablewriter v0.0.0-20160610135559-80b567a11ad5 h1:gmD7q6cCJfBbcuobWQe/KzLsd9Cd3amS1Mq5f3uU1qo=
```

Comment:
Hola. Ok, got it. I'll lead with this: I really want to keep the pretty table in here. Mainly because I like pretty things, secondly because `gpctl` is meant for interaction with `goProbe` in a human-friendly way ☺️ .

But I didn't just pull that dependency out of my rear. I actually got the idea of using it from another large, Cloud-Native project (NATS) and they employ it to print messages. The same has proven quite useful in debugging message buses at work.

As for your points:
1. **maintenance**: I'd agree more strongly with you if this were a dependency deep down in query/packet processing. Given that the nature of `gpctl` is more peripheral, I'm less concerned. Also, since the functionality that we need from the library is completely clear and bounded. Also, it supports both **bold**face and colors, both of which I'm missing in the `tabwriter` library from the std lib.
1. **`grep`-ability**: I have been thinking about introducing a flag `-j, --json` to `gpctl` for exactly this purpose, claiming that `jq > grep` when it comes to extracting information in a meaningful way from the shell. Yes, it would make `gpctl` a glorified `curl`, but so what. It already is
1. **go.mod**: I'd be willing to fork the `tablewriter` project and introduce a `go.mod`. Worst case fix any bugs that come up.

Cheers

Reply:
Thanks for addressing / discussing my points. I'm certainly not gonna put a blocker on this "just because", so by all means, let's keep the thing (after all it at least has no further upstream dependencies, so it relies solely on the Go standard library, which is kept up to date). I like the point about having a flag that allows to trigger "greppable" / plain output though...